prowler/lib/cli/parser.py

```diff
@@ -445,6 +445,12 @@ def __init_azure_parser__(self):
             default=[],
             help="Azure subscription ids to be scanned by prowler",
         )
+        azure_parser.add_argument(
+            "--tenant-id",
+            nargs="+",
+            default=[],
+            help="Azure tenant id to be used with --browser-auth option",
+        )
```

Comment:
Why this variable is a list? If we need to pass a list of tenants please change it to `--tenant-ids` and change the description accordingly.